plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -1357,6 +1357,15 @@ def calculate_size(file_handle):
         # Calculate the size by seeking to the end to find the file size with tell()
         return file_utils.calculate_size(file_handle)
 
+    def get_symlink_name(self):
+        """
+        Provides the name that should be used when creating a symlink.
+        :return: file name as it appears in a published repository
```

Comment:
I usually add a whitespace line between return args, args, and docstring descriptions.